cli/src/main/java/com/devonfw/tools/ide/log/IdeLogLevel.java

```diff
@@ -74,4 +77,16 @@ public boolean isCustom() {
     return (this == STEP) || (this == INTERACTION) || (this == SUCCESS) || (this == PROCESSABLE);
   }
 
+  /**
+   * @param errorHandling the given {@link ProcessErrorHandling}.
+   * @return matching {@link IdeLogLevel}.
+   */
+  public static IdeLogLevel getLogLevelFromErrorHandling(ProcessErrorHandling errorHandling) {
```

Comment:
Please consider SoC and try to put the logic as close as possible to where it belongs to - that would be `ProcessErrorHandling` itself.

Reply:
I've moved this to ProcessErrorHandling now.